app/views/statuses/_poll.html.haml

```diff
@@ -16,7 +16,8 @@
             %span.poll__option__text
               = Formatter.instance.format_poll_option(status, option, autoplay: autoplay)
 
-          %span.poll__chart{ style: "width: #{percent}%" }
+          %progress{ max: 100, value: percent < 1 ? 1 : percent }
```

Comment:
This makes bar accessible by screen reader software, which could have been a good thing if only it wouldn't repeat [the same thing above](https://github.com/tootsuite/mastodon/blob/2c9038b416713bf2793803d16643c37df08e30da/app/views/statuses/_poll.html.haml#L15) and confuse user with its role (*progress* bar). Should it be `aria-hidden` from a11y-relying users?